notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>Change to: <strong>Vertex AI Pipelines: Evaluating BatchPrediction results from a Custom Tabular classification model</strong></p><p>For consistency with the other eval notebooks</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>updated.</p>